src/MatBlazor/Components/MatLink/BaseMatLink.cs

```diff
@@ -0,0 +1,118 @@
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace MatBlazor
+{
+    public class BaseMatLink: BaseMatDomComponent
+    {
+        protected async override Task OnFirstAfterRenderAsync()
+        {
+            await base.OnFirstAfterRenderAsync();
+            await JsInvokeAsync<object>("matBlazor.matButton.init", Ref);
+        }
+
+        public BaseMatLink()
+        {
+            ClassMapper
+                .Add("mdc-button")
+                .Add("mdc-link")
+                .If("mdc-button--raised", () => this.Raised)
+                .If("mdc-button--unelevated", () => this.Unelevated)
+                .If("mdc-button--outlined", () => this.Outlined)
+                .If("mdc-button--dense", () => this.Dense);
+        }
+
+        /// <summary>
+        ///  Event occurs when the user clicks on an element.
```

Comment:
Not a blocker, but the code comments are inconsistent (e.g. here we have more spaces than in other comments.

Reply:
Fixed. (I believe you're referring to the space before ```Event```. Please clarify if it's something else).